che-plugin-openshift-ext-client/src/main/java/org/eclipse/che/ide/ext/openshift/client/project/wizard/page/configure/ConfigureProjectViewImpl.java

```diff
@@ -179,21 +182,31 @@ public void onOpenShiftProjectNameChanged(KeyUpEvent event) {
     @UiHandler({"cheProjectNameInput"})
     public void onCheProjectNameChanged(KeyUpEvent event) {
         if (cheProjectNameInput.getValue().isEmpty()) {
-            setRewriteCheProjectName(true);
+            rewriteCheProjectName=true;
         } else {
-            setRewriteCheProjectName(false);
+            rewriteCheProjectName=false;
         }
         delegate.onCheNewProjectNameChanged();
     }
 
     @UiHandler({"osProjectDescriptionInput"})
     public void onOpenShiftProjectDescriptionChanged(KeyUpEvent event) {
-        cheProjectDescriptionInput.setValue(osProjectDescriptionInput.getValue(), true);
+        if (rewriteCheProjectDescription) {
+            cheProjectDescriptionInput.setValue(osProjectDescriptionInput.getValue(), true);
+        }
+        if (event.getNativeKeyCode() == KeyCodes.KEY_ENTER) {
```

Comment:
Please, can you explain what does this code line do? Why we should not call onOpenShiftDescriptionChanged if KEY_ENTER was pressed?